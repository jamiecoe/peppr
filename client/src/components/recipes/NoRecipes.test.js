import React from 'react';
import { shallow } from 'enzyme';
import NoRecipes from './NoRecipes';

describe('NoRecipes', () => {
  const noRecipes = shallow(<NoRecipes />);

  it('renders properly', () => {
    expect(noRecipes).toMatchSnapshot();
  });
});
