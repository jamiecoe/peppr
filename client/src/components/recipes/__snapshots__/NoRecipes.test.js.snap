// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoRecipes renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoRecipes />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="noRecipes__img"
        style={
                Object {
                        "backgroundImage": "url(no-recipes-img.jpg)",
                      }
        }
>
        <p
                className="noRecipes__message"
        >
                No recipes saved yet!
        </p>
        <Link
                className="noRecipes__btn"
                replace={false}
                to="/addrecipe"
        >
                <span>
                        Add a recipe
                </span>
        </Link>
</div>,
      "className": "noRecipes__container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="noRecipes__message"
>
            No recipes saved yet!
</p>,
          <Link
            className="noRecipes__btn"
            replace={false}
            to="/addrecipe"
>
            <span>
                        Add a recipe
            </span>
</Link>,
        ],
        "className": "noRecipes__img",
        "style": Object {
          "backgroundImage": "url(no-recipes-img.jpg)",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "No recipes saved yet!",
            "className": "noRecipes__message",
          },
          "ref": null,
          "rendered": "No recipes saved yet!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <span>
              Add a recipe
</span>,
            "className": "noRecipes__btn",
            "replace": false,
            "to": "/addrecipe",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Add a recipe",
            },
            "ref": null,
            "rendered": "Add a recipe",
            "type": "span",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="noRecipes__img"
          style={
                    Object {
                              "backgroundImage": "url(no-recipes-img.jpg)",
                            }
          }
>
          <p
                    className="noRecipes__message"
          >
                    No recipes saved yet!
          </p>
          <Link
                    className="noRecipes__btn"
                    replace={false}
                    to="/addrecipe"
          >
                    <span>
                              Add a recipe
                    </span>
          </Link>
</div>,
        "className": "noRecipes__container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="noRecipes__message"
>
              No recipes saved yet!
</p>,
            <Link
              className="noRecipes__btn"
              replace={false}
              to="/addrecipe"
>
              <span>
                            Add a recipe
              </span>
</Link>,
          ],
          "className": "noRecipes__img",
          "style": Object {
            "backgroundImage": "url(no-recipes-img.jpg)",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "No recipes saved yet!",
              "className": "noRecipes__message",
            },
            "ref": null,
            "rendered": "No recipes saved yet!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <span>
                Add a recipe
</span>,
              "className": "noRecipes__btn",
              "replace": false,
              "to": "/addrecipe",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Add a recipe",
              },
              "ref": null,
              "rendered": "Add a recipe",
              "type": "span",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
